data_structures/linked-lists.py

```diff
@@ -0,0 +1,53 @@
+class Node():
+  def __init__(self, value, next_node=None):
+    self.value = value
+    self.next_node = next_node
+
+
+class LinkedList():
+  def __init__(self, head_node_value):
+    self.head_node = Node(head_node_value)
+
+  def add_to_beginning(self, value):
+    new_head = Node(value)
+    new_head.next_node = self.head_node
+    self.head_node = new_head
```

Comment:
Python lets you do these fun swaps in one line like so:

`self.head_node, new_head.next_node = new_head, self.head_node`